net-nds/389-ds-base/389-ds-base-1.4.2.16.ebuild

```diff
@@ -48,12 +44,9 @@ RDEPEND="${COMMON_DEPEND}
 	virtual/perl-Time-Local
 	virtual/perl-MIME-Base64"
 
-pkg_setup() {
-	enewgroup dirsrv
-	enewuser dirsrv -1 -1 -1 dirsrv
-}
-
 src_prepare() {
+	eapply "${FILESDIR}/${PN}-db-gentoo.patch"
+	eapply_user
 	# as per 389 documentation, when 64bit, export USE_64
 	use amd64 && export USE_64=1
```

Comment:
that was for mozldap and thats gone now.
https://bugzilla.redhat.com/show_bug.cgi?id=1255701

Reply:
Right, USE_64 is not used anymore.